src/detect.c

```diff
@@ -1296,6 +1296,46 @@ static inline void StoreDetectFlags(DetectTransaction *tx, const uint8_t flow_fl
     }
 }
 
```

Comment:
If we merge this one, could we have a comment indicating what params j and k are?

Reply:
Good idea, will do. IIRC they are size of both sorted lists to merge into a big one